simulator/modules/core/perf_sim.cpp

```diff
@@ -13,6 +13,12 @@
 namespace config {
     static const AliasedValue<std::string> units_to_log = { "l", "logs", "nothing", "print logs for modules"};
     static const Switch topology_dump = { "tdump", "module topology dump into topology.json" };
+    // command line flag argument for enable/disable prefetch
```

Comment:
This comment does not bring any value compared to code. `AliasedSwitch` is a command line flag argument, and its string description must describe it enough.